enatega-multivendor-app/src/screens/Otp/Phone/PhoneOtp.js

```diff
@@ -47,64 +53,71 @@ function PhoneOtp(props) {
-                otpError && styles().errorInput
-              ]}
-              codeInputHighlightStyle={{
-                borderColor: currentTheme.iconColorPink
-              }}
-              autoFocusOnLoad
-              code={otp}
-              onCodeChanged={code => setOtp(code)}
-              onCodeFilled={code => {
-                onCodeFilled(code)
-              }}
-              editable
-            />
-            {otpError && (
+      <KeyboardAvoidingView>
+        <View style={styles(currentTheme).mainContainer}>
+          <View style={styles().subContainer}>
+            <View style={styles().logoContainer}>
+              <Ionicons name="phone-portrait-outline" size={30} color="black" />
+            </View>
+            <View>
               <TextDefault
-                style={styles().error}
+                H3
+                bolder
+                textColor={currentTheme.fontfourthColor}
+                style={{
+                  ...alignment.MTlarge,
+                  ...alignment.MBmedium
+                }}>
+                {t('verifyPhone')}
+              </TextDefault>
+              <TextDefault
+                H5
                 bold
-                textColor={currentTheme.textErrorColor}>
-                {t('wrongOtp')}
+                textColor={currentTheme.fontSecondColor}
+                style={{
+                  paddingBottom: scale(5)
+                }}>
+                Enter 4 digit code sent to you mobile
               </TextDefault>
-            )}
+              <TextDefault H5 bold textColor={currentTheme.fontfourthColor}>
+                {phone}
+              </TextDefault>
+            </View>
+            <View>
+              <OTPInputView
+                pinCount={6}
+                style={styles().otpInput}
+                codeInputFieldStyle={[
+                  styles().otpBox,
+                  otpError && styles().errorInput
+                ]}
+                codeInputHighlightStyle={{
+                  borderColor: currentTheme.iconColorPink
+                }}
+                autoFocusOnLoad
+                code={otp}
+                onCodeChanged={code => setOtp(code)}
+                onCodeFilled={code => {
+                  onCodeFilled(code)
+                }}
+                editable
+              />
+              {otpError && (
+                <TextDefault
+                  style={styles().error}
+                  bold
+                  textColor={currentTheme.textErrorColor}>
+                  {t('wrongOtp')}
+                </TextDefault>
+              )}
+            </View>
+          </View>
+          <View style={alignment.MBxSmall}>
+            <TextDefault center H4 bold style={alignment.MTsmall}>
+              {seconds !== 0 ? `Retry after ${seconds}s` : ''}
+            </TextDefault>
           </View>
-          <View style={{ ...alignment.MTlarge }}>
+          <View style={{ width: '100%', marginBottom: 20 }}>
```

Comment:
Move it to stylesheet and don't use hard-coded value, instead try to use functions like scale etc.

Reply:
done